docs/ui.go

```diff
@@ -39,10 +39,15 @@ var ui = `
     <script src="https://cdnjs.cloudflare.com/ajax/libs/swagger-ui/3.20.5/swagger-ui-bundle.js"> </script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/swagger-ui/3.20.5/swagger-ui-standalone-preset.js"> </script>
     <script>
+    function getBaseURL() {
+      var path = window.location.pathname
+      path = path.substr(0, path.lastIndexOf('/')+1)
+      return window.location.host + path;
+    }
     window.onload = function() {
       // Begin Swagger UI call region
       const ui = SwaggerUIBundle({
-        url: "../swagger",
+        url: "swagger?base="+encodeURIComponent(getBaseURL()),
```

Comment:
This is a little hacky IMO, but I cant really think of a better way :(.

Reply:
Yeah if we want the try option on swagger to work I think this is the only way.